features/step_definitions/login_and_registration_steps.rb

```diff
@@ -0,0 +1,58 @@
+Given("I am not signed in") do
+  # This step definition intentionally left blank, as being not-signed in is the default
+end
+
+Given("I am signed in") do
+  user = create(:user)
+  app.sign_in_as(user: user)
+end
```

Comment:
Not blocking - I was thinking in a refactor pass of removing this step in favor of the one below to reduce the decision making overhead a bit.